examples/hello-world/hello_world.description

```diff
@@ -1,2 +1,2 @@
-// Let's print "Hello, World!" using Ballerina. 
-// Here's the full source code.
\ No newline at end of file
+// Let's print "Hello, World!" using a `main` function in Ballerina.
+// A public function named `main` is considered the default function as an entry point.
```

Comment:
```suggestion
// A public function named `main` is considered as the default entry point.
```

Reply:
What I was trying to say here was that a `main` function is the default function that would be considered an entry point. Wanted to emphasize this, because we can have other public functions as entry points too.